Corona-Warn-App/src/main/java/de/rki/coronawarnapp/storage/TracingRepository.kt

```diff
@@ -135,9 +135,9 @@ class TracingRepository @Inject constructor(
         )
 
         // check if the keys were not already retrieved today
-        val keysWereNotRetrievedToday =
+        val keysWereNotAlreadyRetrieved =
             LocalData.lastTimeDiagnosisKeysFromServerFetch() == null ||
-                currentDate.withTimeAtStartOfDay() != lastFetch.withTimeAtStartOfDay()
+                currentDate.isAfter(lastFetch)
```

Comment:
Same here, against what case of unwated execution does the check of `currentDate.isAfter(lastFetch)` protect us?

Reply:
see above. Same check.

Reply:
Implemented this differently now. Making sure the task if only run once every hour.